docs/getting-started/install-guide.md

```diff
@@ -29,22 +37,24 @@ The Windows Installer (`.msi` file) is located here: [https://github.com/decred/
 
 `dcrinstall` is an automatic installer and upgrader for the Decred software. The newest release can be found here: [https://github.com/decred/decred-release/releases](https://github.com/decred/decred-release/releases). Binaries are provided for Windows, OSX/macOS, Linux, OpenBSD, and FreeBSD. Executing installer will install `dcrd`, `dcrwallet`, and `dcrctl`. Instructions are provided for Mac, Linux, and Windows below (assumed proficiency for *BSD users).
 
+This method is recommended over the traditional manual installation. `dcrinstall` will automatically download the precompiled, signed binaries found on Github, verify the signatures of the binaries, copy them to a specific folder dependant on OS, create configuration files with settings to allow the 3 applications to communicate with each other, and run you through the wallet creation process. After running through `dcrinstall`, you will be able to launch the software with no additional configuration.
```

Comment:
Technically, the package is what is hashed and that signed, not the individual binaries.  So maybe say verify the package signature instead.